### ISSUE TITLE
[cssom] Expose array of selectors on CSSStyleRule

### ISSUE THREAD

**Issue by bramus** [w3c/csswg-drafts#8999]
`CSSStyleRule` has a `selectorText` property but there seems to be no way to access the individual selectors themselves. This seems like a gap, as authors now need to rely on external libraries to extract the individual selectors from the `selectorText` should it represent a SelectorList.

**Comment by Loirooriol** [comment]
But is providing the individual complex selectors sufficient? Or will authors also want to get the compound selectors of a complex one, or the simple selectors of a compound one?

**Comment by tabatkins** [comment]
Yeah, I'd like to understand the use-cases better here, as I'm not sure why having access to the individual selectors as strings is particularly more helpful than just the current whole list as a string. If the use-cases actually want a parsed representation we should do that instead.

**Comment by bramus** [comment]
For my use-case specifically – in which I analyze the selectors used on a page – I don’t need a fully parsed representation but only the individual selectors. Of course, other people might want more info so yeah, it’s not clear where to draw the line.
